schema/vrs.yaml

```diff
@@ -0,0 +1,582 @@
+
+
+  # =============================================================================
+  # NON-IDENTIFIABLE TYPES
+  # These types have a `type` attribute
+  # =============================================================================
+
+  # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  # SequenceIntervals
+
+  SequenceInterval:
+    additionalProperties: false
+    oneOf:
+      - $ref: '#/definitions/SimpleInterval'
+      - $ref: '#/definitions/NestedInterval'
+    discriminator:
+      propertyName: type
+
+  SimpleInterval:
+    description: >-
+      A SimpleInterval represents a span of sequence. Positions are
+      always represented by contiguous spans using interbase
+      coordinates.
+
+      This definition of SequenceInterval is intended to be compatible with that
+      in Sequence Ontology
+      ([SO:0000001](http://www.sequenceontology.org/browser/current_svn/term/SO:0000001)),
+      with the exception that the GA4GH VRS SequenceInterval may be
+      zero-width. The SO definition is for an "extent greater than
+      zero".
+    additionalProperties: false
+    type: object
+    properties:
+      type:
+        type: string
+        enum: ["SimpleInterval"]
+        default: "SimpleInterval"
+      start:
+        type: ["integer", "null"]
+        default: null
+      end:
+        type: ["integer", "null"]
+        default: null
+    required:
+      - start
+      - end
+    example:
+      type: SimpleInterval
+      start: 11
+      end: 22
+
+  NestedInterval:
+    type: object
+    additionalProperties: false
+    properties:
+      type:
+        type: string
+        enum: ["NestedInterval"]
+        default: "NestedInterval"
+      inner:
+        $ref: '#/definitions/SimpleInterval'
+      outer:
+        $ref: '#/definitions/SimpleInterval'
+    example:
+      type: NestedInterval
+      inner:
+        type: SimpleInterval
+        start: 20
+        end: 30
+      outer:
+        type: SimpleInterval
+        start: 10
+        end: 40
+
+  CytobandInterval:
+    description: >-
+      A contiguous region specified by chromosomal bands features.
+    type: object
+    additionalProperties: false
+    properties:
+      type:
+        type: string
+        enum: [ "CytobandInterval" ]
+        default: "CytobandInterval"
+      start:
+        $ref: "#/definitions/HumanCytoband"
+      end:
+        $ref: "#/definitions/HumanCytoband"
+    required:
+      - start
+      - end
+    example:
+      type: CytobandInterval
+      start: q22.2
+      end: q22.3
+
+
+  # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  # SequenceExpression
+
+  SequenceExpression:
+    description: >-
+      One of a set of sequence representation syntaxes.
+    additionalProperties: false
+    oneOf:
+      - $ref: "#/definitions/LiteralSequence"
+      - $ref: "#/definitions/DerivedSequence"
+      - $ref: "#/definitions/RepeatedSequence"
```

Comment:
Rename these classes to `LiteralSequenceExpression`, `DerivedSequenceExpression`, and `RepeatedSequenceExpression`, respectively.

Reply:
Done in be7e150